app/Http/Controllers/ApiLicenseController.php

```diff
@@ -34,10 +34,10 @@ public function create()
 	 *
 	 * @return Response
 	 */
-	public function store()
-	{
-		$license = ActCode::create(Request::all());
-		return response()->json($licenses);
+	public function store(Request $request)
```

Comment:
request is now supported, returns the added license (as JSON) if store is successful